daemon/main.go

```diff
@@ -115,6 +115,11 @@ func init() {
 						Name:        "k",
 						Usage:       "Kubernetes master address server",
 					},
+					cli.BoolTFlag{
+						Destination: &config.KeepConfig,
+						Name:        "keep-config",
+						Usage:       "When restoring state, keeps containers' configuration in place",
```

Comment:
I think state is  misleading wording for this in general. We should rename that option as well and simply call it `--restore`.